snippets/csharp/programming-guide/ref-returns/NumberStore.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace RefReturns
+{
+
+    // <Snippet1>
+    using System;
+
+    class NumberStore
+    {
+        int[] numbers = { 1, 3, 7, 15, 31, 63, 127, 255, 511, 1023 };
+
+        public ref int FindNumber(int target)
+        {
+            for (int ctr = 0; ctr < numbers.Length; ctr++)
+            {
+                if (numbers[ctr] >= target)
+                    return ref numbers[ctr];
+            }
+            return ref numbers[0];
+        }
+
+        public override string ToString()
+        {
+            string retval = "";
+            for (int ctr = 0; ctr < numbers.Length; ctr++)
+            {
+                retval += $"{numbers[ctr]} ";
+            }
+            return retval.Trim();
```

Comment:
I see that is pre-existing code, but to make snippet shorter what about such `ToString()` implementation:
```csharp
public override string ToString() => string.Join(" ", numbers);
```

Same remark also for the *NumberStoreUpdated.cs* file